### ISSUE TITLE
Add <title> tag to popup.html

### ISSUE THREAD

**Issue by brendanjerwin** [brendanjerwin/simple_wiki#397]
SonarCloud reports a MAJOR bug: popup.html (extensions/online-order-recorder/popup.html line 3) is missing a `<title>` tag. This is an accessibility requirement (WCAG 2.4.2).

> [!tip]
> Copilot, please fix this.
